### PR TITLE
Fix simulation stopping at unhandled messages in `DBFSAlgorithm`.

### LINKED ISSUE #821: Fix simulation stopping at unhandled messages in `DBFSAlgorithm`
There seems to be an issue in the way the `DBFSAlgorithm` handles `unhandled` messages.

image

### PR DESCRIPTION
**Pull request by staudtMarius**
Resolves #821

### CODE REVIEW

File: src/main/scala/edu/ie3/simona/agent/grid/DBFSAlgorithm.scala [hunk 1]
```diff
@@ -802,9 +803,10 @@ trait DBFSAlgorithm extends PowerFlowSupport with GridResultsSupport {
           buffer.stash(powerResponse)
           Behaviors.same
 
-        case _ =>
-          // preventing "match may not be exhaustive"
-          Behaviors.unhandled
+        case (message, _) =>
+          ctx.log.debug(s"Received the message $message to early. Stash away!")
```
Comment (sebastian-peter) [inline src/main/scala/edu/ie3/simona/agent/grid/DBFSAlgorithm.scala:807]: same as above

**Review by sebastian-peter — CHANGES_REQUESTED** [review]
Also question from my side: Was this an issue that also existed before the transition to pekko typed?

**Comment by staudtMarius** [comment]
@sebastian-peter I never noticed this issue before the conversion to typed.

**Comment by sebastian-peter** [comment]
@staudtMarius Then let's analyze what the problem is and where it comes from first before merging this

**Comment by sebastian-peter** [comment]
@staudtMarius We still have a todo here if I remember correctly, which was specifying the messages that are supposed to be stashed

**Review by sebastian-peter — APPROVED** [review]
Thanks, looks good

**Pull request merged**